.github/workflows/compile-examples.yml

```diff
@@ -47,6 +49,27 @@ jobs:
           - fqbn: arduino:mbed_opta:opta
             platforms: |
               - name: arduino:mbed_opta
+        include:
+          - board:
+            - fqbn: arduino:mbed_portenta:envie_m7
+              additional-sketch-paths: |
+                - examples/SimpleStorageWriteRead
+                - examples/PortentaH7Logger
+                - examples/AdvancedUSBInternalOperations
+                - examples/BackupInternalPartitions
+          - board:
+            - fqbn: arduino:renesas_portenta:portenta_c33
+              additional-sketch-paths: |
+                - examples/SimpleStorageWriteRead
+                - examples/AdvancedUSBInternalOperations
+                - examples/BackupInternalPartitions
```

Comment:
I notice these three sketches are compiled for all the boards. If the are intended to be compatible with any of the boards supported by the library, the paths can be moved to `env.UNIVERSAL_SKETCH_PATHS`